bramkouwen/Rock-Paper-Scissors-Game
# Updating README File

## What does this PR do?
* It Updates the README file to contain more detailed information

## How do I check this PR?
* Pull down the branch 
* See changes to Readme File 

## Screenshots 
None.
